packages/forms/src/widgets/DatalistWidget/DatalistWidget.js

```diff
@@ -135,7 +87,7 @@ class DatalistWidget extends React.Component {
 		this.inputProps = {
 			placeholder: props.placeholder,
 			required: props.required,
-			onBlur: event => this.onBlur(event),
+			onBlur: () => this.onBlur(),
```

Comment:
Because it is not used anymore. It was used like that : `const { value } = event.target;` ; but value is never defined.